turns axolotls into basic mobs

kawoppi opened the pull request:
## About The Pull Request
Changes axolotls from simple mobs to basic mobs. I'm not well versed on the differences between simple and basic mobs but it looks like axolotls are still working as before. Their idle movement may be slightly different compared to before and now they don't move away while you're pulling them.
## Why It's Good For The Game
Basic mobs are better for AI and probably some other stuff as well and there's the freeze thing also.
## Changelog
:cl:
code: turned axolotls into basic mobs, their idle movement may be slightly different and they won't move while you're pulling them anymore
/:cl:

kawoppi committed:
added basic axolotl

kawoppi committed:
removed simple axolotl

kawoppi committed:
slightly organized axolotl attributes

tralezab commented:
looks good

CapybaraExtravagante commented on code/modules/mob/living/basic/vermin/axolotl.dm:
```diff
@@ -9,24 +9,33 @@
-/mob/living/simple_animal/axolotl/Initialize(mapload)
+	ai_controller = /datum/ai_controller/basic_controller/axolotl
+
+/mob/living/basic/axolotl/Initialize(mapload)
```
you need to make sure you add the components/elements that makes them breathe air and makes them be damaged by temperature :)

kawoppi replied:
Good catch! Did some testing on it and found out that none of the current basic mobs will die in space, except for stickmen. I'll fix it for axolotls tomorrow. It might be better to make that the default for basic mobs unless overridden, like it was with simple mobs. Or maybe it was left out for a reason

kawoppi committed:
makes basic axolotls die from bad temperatures/atmos like they used to

The atmos values match the ones used by the default simplemob. The minimum and maximum body temperature and damage values in the default basic_body_temp_sensitive element match those set on the default simplemob as well.
